packages/ndla-forms/src/InputV3.tsx

```diff
@@ -102,10 +102,13 @@ export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>((props, r
   useEffect(() => {
     window.addEventListener("input", resize);
     window.addEventListener("resize", resize);
+    const textareaRef = localRef.current;
+    if (textareaRef) textareaRef.addEventListener("click", resize);
```

Comment:
Kan vi bruke `?.addEventListener` her?

Reply:
Ja, akkurat det samme hvilken variant vi går for, jeg syntes kanskje if-sjekken er litt mer leselig her .. 🤔